pkg/controller/node/ipam/sync/sync.go

```diff
@@ -173,7 +173,7 @@ func (sync *NodeSync) Delete(node *v1.Node) {
 
 // syncOp is the interface for generic sync operation.
 type syncOp interface {
-	// run the requested sync operation.
+	// Run the requested sync operation.
```

Comment:
actually go want this to be lowercase `run`

Reply:
Thanks! Reverted the change. PTAL